cosmos/lib/cosmos/api/tlm_api.rb

```diff
@@ -296,14 +192,12 @@ def get_tlm_buffer(target_name, packet_name, scope: $cosmos_scope, token: $cosmo
     #
     # @param target_name [String] Name of the target
     # @param packet_name [String] Name of the packet
-    # @param value_type [Symbol] How the values should be converted. Must be
-    #   one of {Packet::VALUE_TYPES}
+    # @param type [Symbol] Types returned, :RAW, :CONVERTED (default), :FORMATTED, or :WITH_UNITS
     # @return (see Cosmos::Packet#read_all_with_limits_states)
-    def get_tlm_packet(target_name, packet_name, value_type = :CONVERTED, scope: $cosmos_scope, token: $cosmos_token)
+    def get_tlm_packet(target_name, packet_name, type: :CONVERTED, scope: $cosmos_scope, token: $cosmos_token)
```

Comment:
I converted the value_type to type: keyword everywhere to be consistent with the rest of the API.